app/inbound/calculation/get-calculation-by-sbi.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../../data')
+
+const getCalculationBySbi = async (sbi, transaction) => {
```

Comment:
This function is called get calculation but returns an organisation?

Reply:
![image](https://user-images.githubusercontent.com/56440327/187409475-5deea806-746d-4a90-8273-f8bd39397004.png)

Fixing going to get by `calculationReference` to clear up comment below too

Reply:
Hmmm, GitHub dnt do gifs... noted